### ISSUE TITLE
Review implementation support - SVG 2 Master issue

### ISSUE THREAD

**Issue by AmeliaBR** [w3c/svgwg#487]
Single issue collecting issues discussing SVG 2 changes & implementation:

By chapter:

- [ ] 1, Introduction
- [ ] 2, Conformance Criteria
- [ ] 3, Rendering Model
- [ ] 4, Basic Data Types
- [ ] 5, Document Structure
- [ ] 6, Styling
- [ ] 7, Geometry Properties
- [ ] 8, Coordinate Systems, Transformations, and Units
- [ ] 9, Paths #385
- [ ] 10, Basic Shapes #484
- [ ] 11, Text
- [ ] 12, Embedded Content #384
- [ ] 13, Painting (fill, stroke, markers)
- [ ] 14, Paint servers
- [ ] 15, Scripting and Interactivity
- [ ] 16, Linking
- [ ] Appendices

(I'll add links to the chapter issues as I create them.)

Each chapter's implementation issue will have a checklist of new features, which should not be checked off until either:

- tests have been contributed, and at least 2 implementations pass (and ideally, bugs filed on any non-passing browsers/major authoring tools)
- or, the feature is removed or deferred

Only once all the features in a chapter's implementation checklist have been checked off, that chapter can be checked off in this issue.
